indexer/services/ender/src/scripts/handlers/dydx_open_interest_update_handler.sql

```diff
@@ -0,0 +1,39 @@
+CREATE OR REPLACE FUNCTION dydx_open_interest_update_handler(event_data jsonb) RETURNS jsonb AS $$
+/**
+  Parameters:
+    - event_data: The 'data' field of the IndexerTendermintEvent (https://github.com/dydxprotocol/v4-chain/blob/9ed26bd/proto/dydxprotocol/indexer/indexer_manager/event.proto#L25)
+        converted to JSON format. Conversion to JSON is expected to be done by JSON.stringify.
+  Returns: JSON object containing fields:
+    - open_interest_update: The updated perpetual market open interest
+
+  (Note that no text should exist before the function declaration to ensure that exception line numbers are correct.)
+*/
+DECLARE
+    perpetual_market_record perpetual_markets%ROWTYPE;
+    updates_array jsonb[];
+    open_interest_update jsonb;
+BEGIN
+    FOR open_interest_update IN SELECT * FROM jsonb_array_elements(event_data->'openInterestUpdates') LOOP
+      perpetual_market_record."id" = (open_interest_update->'perpetualId')::bigint;
+      perpetual_market_record."openInterest" = dydx_from_serializable_int(open_interest_update->'openInterest');
```

Comment:
it seems like openInterest from protocol would come in quantums, but you would want to save open interest into `perpetual_markets` in USDC

Reply:
This is in base quantums, we need oracle price to convert it to usdc from my understanding

Reply:
This is completely different from how we use this field today. Is front end aware of this? Today, indexer does not share any quantum subtick etc data to the FE, all data is in human readable format.
 - is FE aware that you are sharing this in base quantums?
- can we at the very least store this in human readable base asset? Ie # of bitcoin?
- if we store this in base asset, we should use a new field "baseOpenInterest"

Reply:
1. yes FE is aware of this
2, 3. I added a field to store in human readable base asset and named it baseOpenInterest